build.py

```diff
@@ -85,10 +85,10 @@ def calc_spki_hash(cert):
             stdin = subprocess.PIPE,
             stdout = subprocess.PIPE)
     stdout, _ = proc.communicate(cert.encode('utf-8'))
-    stdout = stdout.decode('utf-8')
+    stdout = stdout.decode('utf-8').lower()
     assert proc.returncode == 0
-    assert stdout.startswith('SHA256 Fingerprint=')
-    hash = stdout.replace('SHA256 Fingerprint=', '').replace(':', '')
+    assert stdout.startswith('sha256 fingerprint=')
+    hash = stdout.replace('sha256 fingerprint=', '').replace(':', '')
```

Comment:
I had made a similar fix in https://github.com/rustls/webpki-roots/pull/26 but then ended up reverting it because the `openssl` in CI was still using all-caps. 

I guess since this builds green CI's openssl is matching the expected as lower-case now? :+1:

Reply:
Ahhh! it's the `lower()` call you added that does the trick. Nevermind me :-)